### ISSUE TITLE
Plugin deletes a "<" when using `cit` and tag is empty

### ISSUE THREAD

**Issue by nandryshak** [paradigm/TextObjectify#3]
Say I have an HTML tag like this:

`<p></p>`

Put the cursor anywhere on the tag and type `cit` (change inner tag). You get this:

`<p>|/p>`

where `|` is the cursor.

**Comment by paradigm** [comment]
That's definitely a bug.  It seems TextObjectify isn't acting properly when there is nothing between the delimiters.  I'll look into it.

**Comment by nandryshak** [comment]
Thanks. Also if there's a space (like this `<p> </p>`) and you do `cit` it deletes the whole tag. Two spaces works fine,  as well as a new line. That is, `cit` on

```
<p>
</p>
```

works as expected.

**Comment by paradigm** [comment]
Fixed in dacfa7b6c6eef75397eb36e9a1d0c8e3f6f8de42

**Comment by nandryshak** [comment]
It seems as though the original issue is persistent (Plugin deletes a "<" when using `cit` and tag is empty). But this: (Also if there's a space (like this <p> </p>) and you do cit it deletes the whole tag) looks fixed.

**Comment by paradigm** [comment]
Yeah, you're right.  I'm not sure why I thought this was fixed.  I appreciate you commenting again on it.  I'm a bit swamped at the moment with other priorities, but I'll get back to this within the next month or so and be sure I fix it correctly this time.

**Comment by ChanderG** [comment]
I have a hack to work around this [here](https://github.com/ChanderG/TextObjectify/commit/142040f5dbd9b4464a870deec53829a39a871c41). I can open a PR if there is any interest.
